tests/src/Unit/Command/CreateEdgeRoleCommandTest.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\apigee_edge\Unit {
+
+  use Drupal\apigee_edge\CliServiceInterface;
+  use Drupal\apigee_edge\Command\CreateEdgeRoleCommand;
+  use Drupal\Console\Core\Style\DrupalStyle;
+  use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+  use Drupal\Core\Logger\LogMessageParserInterface;
+  use Drupal\Tests\UnitTestCase;
+  use Prophecy\Argument;
+  use Symfony\Component\Console\Formatter\OutputFormatterInterface;
+  use Symfony\Component\Console\Input\InputInterface;
+  use Symfony\Component\Console\Output\OutputInterface;
+
+  /**
+   * Test ApigeeEdgeCommands class.
+   *
+   * @group apigee_edge
+   */
+  class CreateEdgeRoleCommandTest extends UnitTestCase {
+
+    /**
+     * The system under test.
+     *
+     * @var \Drupal\apigee_edge\Command\CreateEdgeRoleCommand
+     */
+    protected $createEdgeRoleCommand;
+
+    /**
+     * The CLI Service mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $cliService;
+
+    /**
+     * The IO mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $io;
+
+    /**
+     * The LogMessageParser mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $logMessageParser;
+
+    /**
+     * The LoggerChannelFactory mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $loggerChannelFactory;
+
+    /**
+     * The InputInterface mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    private $input;
+
+    /**
+     * The OutputInterface mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    private $output;
+
+    /**
+     * OutputFormatterInterface mock.
+     *
+     * @var \Prophecy\Prophecy\ObjectProphecy
+     */
+    private $outputFormatter;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp() {
+      parent::setUp();
+      $this->cliService = $this->prophesize(CliServiceInterface::class);
+      $this->logMessageParser = $this->prophesize(LogMessageParserInterface::class);
+      $this->loggerChannelFactory = $this->prophesize(LoggerChannelFactoryInterface::class);
+      $this->createEdgeRoleCommand = new CreateEdgeRoleCommand($this->cliService->reveal(),
```

Comment:
Running this test locally gives me an error because Edge does not require Drupal Console. Trying it out in travis right now, but I think we should at least add Drupal Console to `.travis/composer.json`.

Reply:
I will also add drush which is missing